tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -622,7 +623,7 @@ async function cleanCompassResourcesSKR(client, appName, scenarioName, runtimeID
   }
 }
 
-async function ensureCommerceMockLocalTestFixture(mockNamespace, targetNamespace) {
+async function ensureCommerceMockLocalTestFixture(mockNamespace, targetNamespace, testV1Alpha2CRD=false) {
```

Comment:
What about testSubscriptionV1Alpha2 instead of testV1Alpha2CRD?

Reply:
I have addressed you comment.